nuntium/api.py

```diff
@@ -34,11 +35,45 @@ def get_offset_from_page(self):
         return offset
 
 
+class IdentifierResource(ModelResource):
+    class Meta:
+        queryset = Identifier.objects.all()
+        resource_name = 'identifier'
+        filtering = {
+            'identifier': ALL,
+            'scheme': ALL,
+        }
+        allowed_methods = []
+
 class PersonResource(ModelResource):
+    identifiers = fields.ToManyField(IdentifierResource, 'identifiers', full=True)
+
     class Meta:
         queryset = PopoloPerson.objects.all()
         resource_name = 'person'
         authentication = ApiKeyAuthentication()
+        filtering = {
+            'identifiers': ALL_WITH_RELATIONS,
+        }
+
+    def obj_get_list(self, bundle, **kwargs):
+        result = super(PersonResource, self).obj_get_list(bundle, **kwargs)
+
+        filters = bundle.request.GET.copy()
+        if 'contactable' in filters:
+            # Count the number of contacts the person has
+            if filters['contactable'] == 'True':
+                result = result.has_contacts()
+            elif filters['contactable'] == 'False':
```

Comment:
can this either be case-insensitive or stick to one case, rather than python boolean case?